compiler/luci/import/src/GraphBuilder.cpp

```diff
@@ -60,16 +59,19 @@ CircleNode *GraphBuilder::build(const circle::OperatorT &op, GraphBuilderContext
   // Set up node parameters.
   assert(outputs.size() == 1);
   {
-    const circle::TensorT &output_tensor = *tensors[outputs[0]];
+    const auto output_tensor = tensors[outputs[0]];
+    assert(output_tensor != nullptr);
+
```

Comment:
here and below, there are lots of this kind of adding lines.
I don't think there is one correct way but it would be better not to insert these on this title.
they enhance readability and I personally like these but like others, the seem to distract reading changes.

Reply:
Okay, I will reduce this kind of changes too.